examples/splines/spline.ipynb

```diff
@@ -4,16 +4,12 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">that are tied togehter at their boundaries
                  ☝
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/239/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>